### PR TITLE
Added link for crowfunding

### PR DESCRIPTION
**Pull request by SeqLaz**
Sorry for the noise, this PR is just for the restoring the last discussion in the other PR

### CODE REVIEW

**Comment by m-kuhn** [comment]
Is this a duplicate of #405 ?

File: documentation/get-started/sponsor.en.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ The best way to financially support QField is by subscribing to one of the month
 
 ## Crowdfundings
 
-You can be part of an exciting opportunity of something truly impactful! We believe that positive change starts with a collective effort, and we are inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
+You can be part of an exciting opportunity of something truly impactful! We believe that positive change starts with a collective effort, and we are inviting you to join us on this inspiring journey. To turn this vision into reality, we need your support through crowdfunding. Your contribution plays a crucial role in reaching the funding goal and making QField even better.
 
-Please visit our crowdfunding page [Crowdfunding](https://www.opengis.ch/category/crowdfunding/)
+Please visit our [crowdfunding](https://www.opengis.ch/category/crowdfunding/) page.
```
Comment (m-kuhn) [inline documentation/get-started/sponsor.en.md:18]:
```suggestion
Please visit our [crowdfunding page](https://www.opengis.ch/category/crowdfunding/).
```

**Comment by m-kuhn** [comment]
Actually it contains more with the extra link. 👌

File: documentation/get-started/sponsor.en.md (hunk 1 above)
Comment (SeqLaz) [reply documentation/get-started/sponsor.en.md:18]: Ouch, I didn't see it, and yes is duplicated

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**